docs/io/visualization/montecarlo_packet_visualization.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>This is not the correct syntax for a note. See other notebooks in the documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
done